components/inputs/test/input-color.visual-diff.js

```diff
@@ -32,14 +32,14 @@ describe('d2l-input-color', () => {
 	].forEach((name) => {
 		it(name, async function() {
 			const rect = await visualDiff.getRect(page, `#${name} > d2l-input-color`);
-			await visualDiff.screenshotAndCompare(page, this.test.fullTitle(), { clip: rect });
+			await visualDiff.screenshotAndCompare(page, this.test.fullTitle(), { clip: rect, captureBeyondViewPort: false });
```

Comment:
These tests just started suddenly timing out in this particular PR... not locally, not in any of the others. No idea what's going on, but trying the magical fix.